ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly.yaml

```diff
@@ -4648,32 +4648,20 @@ tests:
     test:
     - chain: openshift-e2e-test-hypershift-qe-mgmt-longrun
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-mce
-- as: aws-ipi-ovn-hypershift-mce-n2minor-mgmt-f28
```

Comment:
Looks like something wrong.  Another job got removed. I guess this is unexpected.

Reply:
fixed